fluent_comments/locale/sk/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,169 @@
+msgstr "Moderácia"
+
+#: admin.py:62
+msgid "Hierarchy"
+msgstr "Hierarchia"
+
+#: admin.py:77
+msgid "Page"
+msgstr "Stránka"
+
+#: admin.py:92
+msgid "user's name"
+msgstr "používateľské meno"
+
+#: templates/comments/comment.html:21 templates/comments/preview.html:12
+msgid "Preview of your comment"
+msgstr "Náhľad komentára"
+
+#: templates/comments/comment.html:24
+msgid "Anonymous"
+msgstr "Anonym"
+
+#: templates/comments/comment.html:27
+#, python-format
+msgid "on %(submit_date)s"
+msgstr "poslal %(submit_date)s"
+
+#: templates/comments/comment.html:28
+msgid "moderated"
+msgstr "moderované"
+
+#: templates/comments/comment.html:29
+msgid "reply"
+msgstr "odpovedať"
+
+#: templates/comments/deleted.html:4
+msgid "Thanks for removing"
+msgstr "Ďakujeme za odstránenie"
+
+#: templates/comments/deleted.html:12
+msgid "Thanks for removing the comment"
+msgstr "Ďakujeme za odstranění komentára"
+
+#: templates/comments/deleted.html:14 templates/comments/flagged.html:14
+msgid ""
+"\n"
+"      Thanks for taking the time to improve the quality of discussion on our "
+"site.<br/>\n"
+"      You will be sent back to the article...\n"
+"    "
+msgstr ""
+"\n"
+"      Vďaka, že ste si našli čas na zlepšenie kvality diskusie na našom "
+"webe.<br/>\n"
+"      Budete presmerovaný späť na diskusiu...\n"
+"    "
+
+#: templates/comments/deleted.html:20 templates/comments/flagged.html:20
+#: templates/comments/posted.html:24
+msgid "Back to the article"
+msgstr "Naspäť k diskusii"
+
+#: templates/comments/flagged.html:4
+msgid "Thanks for flagging"
+msgstr "Děkujeme za označkování"
+
+#: templates/comments/flagged.html:12
+msgid "Thanks for flagging the comment"
+msgstr "Vďaka za označenie komentára"
+
+#: templates/comments/form.html:4
+msgid "Comments are closed."
+msgstr "Diskusia je uzavretá."
+
+#: templates/comments/form.html:13 templates/comments/preview.html:26
+msgid "Post"
+msgstr "Odoslať"
+
+#: templates/comments/form.html:14 templates/comments/preview.html:27
+msgid "Preview"
+msgstr "Náhľad"
+
+#: templates/comments/posted.html:3
+msgid "Thanks for commenting"
+msgstr "Vďaka za komentár"
+
+#: templates/comments/posted.html:11
+msgid "Thanks for posting your comment"
+msgstr "Vďaka za komentár"
+
+#: templates/comments/posted.html:13
+msgid ""
+"\n"
+"      We have received your comment, and posted it on the web site.<br/>\n"
+"      You will be sent back to the article...\n"
+"      "
+msgstr ""
+"\n"
+"      Dostali sme váš komentár a zobrazili sme ho na stránke.<br/>\n"
+"      Budete presmerovaný späť na diskusiu...\n"
+"      "
+
+#: templates/comments/preview.html:3
+msgid "Preview your comment"
+msgstr "Náhľad komentára"
+
+#: templates/comments/preview.html:10
+msgid "Please correct the error below"
+msgid_plural "Please correct the errors below"
+msgstr[0] "Prosím opravte nasledujúcu chybu"
+msgstr[1] "Prosím opravte nasledujúce chyby"
+
+#: templates/comments/preview.html:17
+msgid "Post your comment"
+msgstr "Odoslať komentář"
```

Comment:
As @ZoltanOnody pointed out, the correct form would be „Odoslať komentár“ without the caron on “r”.